packages/opencensus-core/src/trace/config/types.ts

```diff
@@ -70,4 +72,28 @@ export interface TracingConfig {
   logger?: Logger;
 }
 
+/** Specify the deafault Tracer Paramaters */
```

Comment:
Can this commented code be removed?

Reply:
I saw this was marked as resolved - is the commented out code below intentional? (Or maybe just a commit not pushed yet?)